berth_reservations/tests/conftest.py

```diff
@@ -38,6 +27,44 @@ def superuser():
     return user
 
 
+@pytest.fixture
+def user_api_client():
+    return create_api_client(user=UserFactory())
+
+
+@pytest.fixture
+def staff_api_client():
+    return create_api_client(user=UserFactory(is_staff=True))
+
+
+@pytest.fixture
+def superuser_api_client():
+    return create_api_client(user=UserFactory(is_superuser=True))
+
+
+@pytest.fixture
+def api_client(request, user_api_client, staff_api_client, superuser_api_client):
```

Comment:
Is it actually ever called with parameters?

Reply:
Oh wait, I got it. Nvm.

Reply:
? we call it now like this:
```python
@pytest.mark.parametrize(
    "api_client", ["api_client", "user_api_client", "staff_api_client"], indirect=True
)
```
in many places. as we agreed 🙂